### DISCUSSION TITLE
my aside not showing the content "on this page"

### DISCUSSION THREAD

**Discussion by Fanreza** [vuejs/vitepress#1375] [Q&A] [answered]
i have a markdown like this

![code](https://user-images.githubusercontent.com/57042204/191409879-eb6f9f24-e294-4630-848e-324b083fcb2f.png)

but its not showing a aside, is there a special configuration for this? i read a aside configuration in fronntmatter, it said aside default is true.

and i cant find a similar discussion or question

thanks for helping.

**Comment by brc-dd** [comment, marked as answer]
Yeah it should be visible by default. Can you share a repository where you're getting this problem? Also, what's your VitePress version?

**Reply by Fanreza** [reply]
I'm using latest version 1.0.0-alpha.15

**Comment by Fanreza** [comment]
i try to deleting some code that may be causing it hidden, and i figured out that `outline: 3;` make the "on this page" hidden, but i dont know why it does? can you please explain to me? i read the docs but still dont understand and there no much tutorial for this

**Reply by brc-dd** [reply]
outline: 3 means that it will only show h3 headings in outline. To show both h2 and h3, you need outline: [2, 3]

**Reply by Fanreza** [reply]
ahh i see so number in outline it means a heading type? and the maximum value is 6 because max heading is h6?

thanks for help and explanation
